rxjs-1/main-grand-final.js

```diff
@@ -0,0 +1,28 @@
+// this race needs 16 listeners to run so we increase defaultMaxListeners
+require('events').EventEmitter.defaultMaxListeners = 16;
```

Comment:
Have you ideas on how this could be avoided thanks to RxJS? I saw that you talked about that in the `Strengths and weaknesses` paragraph, we could discuss that during our next call.

Reply:
I guess multicast is the path to follow, i already succeeded in halving the listeners usage just by using share() on the race observable.
I'm missing knowledge here (and time, to study this feature in details), but i guess it can drop to 2 for the whole challenge ('data' and 'end') by implementing an appropriate Subject.